config/config.yaml.dist

```diff
@@ -267,8 +267,27 @@ dex:
 #
 #        charts:
 #            clusterAutoscaler:
-#                chart: "banzaicloud-stable/cluster-autoscaler"
-#                version: "0.12.3"
+#                chart: "stable/cluster-autoscaler"
+#                version: "6.2.0"
+#                imageVersions:
+#                    - k8sVersion: "1.12"
```

Comment:
Versions do not match with the configured defaults.

Reply:
fixed